### PR TITLE
Migrations 2

### PR DESCRIPTION
**Pull request by brennanjl**
Using this as the PR for updating the migrations tutorial.

### CODE REVIEW

File: docs/node/migrations/zero-downtime-migrations/index.mdx [hunk 1]
```diff
@@ -122,4 +122,4 @@ Conflict resolution principles for account spends:
 
 ## Migration Ends
 
-The old network halts after the `Duration` blocks since the start of the migration. The old network will no longer accept new transactions or produce new blocks with transactions. The RPC services remain available for querying the state of the network. Once the new network is fully synced with the old network, the old network can be safely decommissioned. The new network continues to process transactions and produce blocks as usual.
\ No newline at end of file
+The old network halts after the `Duration` blocks since the start of the migration. The old network will no longer accept new transactions or produce new blocks with transactions. The RPC services remain available for querying the state of the network. Once the new network is fully synced with the old network, the old network can be safely decommissioned. The new network continues to process transactions and produce blocks as usual.
```
Comment (brennanjl) [inline docs/node/migrations/zero-downtime-migrations/index.mdx:125]: I used an online diff checker to ensure that these are the same. IDK why git is saying these are different

**Comment by brennanjl** [comment]
Once someone thinks this is good to go, I will overwrite the old tutorial

File: docs/node/migrations/2-zero-downtime-migrations/index.mdx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+Once >2/3 of the validators approve the migration proposal, the migration will start after the activation period. During this activation period, the network will
+reject any new migration proposals. The activation period allows all network participants to prepare for the upcoming migration.
+
+### Setting Up The New Network
+
+Validators should start setting up their nodes for the new network as soon as possible. All validator should agree on the genesis configuration for the new network except
+the `app_hash`, `validators` and `migration` configurations. These configurations should be left empty as they will be filled in during the migration phase automatically
+by the new nodes when run in `migration` mode. For information on how to automatically generate a genesis file that can be used for a network migration, see the
+[tutorial](/docs/node/migrations/zero-downtime-migrations/tutorial#3.-setting-up-the-new-nodes).
```
Comment (charithabandi) [inline docs/node/migrations/2-zero-downtime-migrations/index.mdx:98]: broken link

**Review by charithabandi — APPROVED** [review]
lgtm!

**Comment by brennanjl** [comment]
Will wait for Luke's go-ahead before merging

**Pull request merged**